librocksdb_sys/src/lib.rs

```diff
@@ -16,6 +16,7 @@
 extern crate libc;
 #[cfg(test)]
 extern crate tempdir;
+extern crate bzip2_sys;
```

Comment:
What's this for?

Reply:
bzip2 is compiled with stdio disabled, so there has to be an external hook to handle error, which is defined in bzip2_sys already.